src/app/pages/ArticlePage/index.test.jsx

```diff
@@ -503,3 +503,19 @@ it('should render PodcastPromos when passed', async () => {
 
   expect(getByText('Что это было?')).toBeInTheDocument();
 });
+
+it('should render Riddle component when passed', async () => {
+  const pageDataWithRiddle = {
+    ...articleDataNews,
+  };
+  const { container } = render(
+    <Context service="russian">
+      <ArticlePage pageData={pageDataWithRiddle} />
+    </Context>,
+  );
+  const actual = container.querySelector(
+    'iframe[src="https://www.riddle.com/embed/a/SAVstNdh?lazyImages=true&staticHeight=false"]',
+  );
+  expect(actual).toBeInTheDocument();
+  expect(container).toMatchSnapshot();
```

Comment:
Do we really need to snapshot the entire container here - is it enough to just check that the riddle component is in the document?